crates/holochain_types/src/app.rs

```diff
@@ -966,6 +966,9 @@ pub enum StoppedAppReason {
 
     /// Same meaning as [`InstalledAppInfoStatus::Disabled`](https://docs.rs/holochain_conductor_api/0.0.33/holochain_conductor_api/enum.InstalledAppInfoStatus.html#variant.Disabled).
     Disabled(DisabledAppReason),
+
+    /// Same meaning as [`InstalledAppInfoStatus::AwaitingMemProofs`](https://docs.rs/holochain_conductor_api/0.0.33/holochain_conductor_api/enum.InstalledAppInfoStatus.html#variant.AwaitingMemProofs).
```

Comment:
Isn't this referring to a really old Holochain version? I'd prefer that the docs from that link were copied here if they're still relevant